decidim.gemspec

```diff
@@ -20,7 +20,15 @@ Gem::Specification.new do |s|
   s.summary = "Citizen participation framework for Ruby on Rails."
   s.description = "A generator and multiple gems made with Ruby on Rails."
 
-  s.files = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^decidim-.*/}) }
+  s.files = Dir[
+    "{docs,lib}/**/*",
+    "Gemfile",
+    "Gemfile.lock",
+    "LICENSE-AGPLv3.txt",
+    "Rakefile",
+    "README.md"
+  ]
```

Comment:
This fixes another problem I run into. In the generator specs, `rake install` would behave unexpectedly because the `.git` folder is not in the docker build context, so it would not be able to find the list of files and the generated gem would be empty. That would cause require errors in the generator. See the commit message for another benefits of this approach: https://github.com/decidim/decidim/pull/2217/commits/ff438756a864697b3d9ebfbbd7eb58a298194405.

I think the reason your PR is not running into this might be that `decidim/docker` does not specify anything in `.dockerignore` so the `.git` folder is actually copied into the image? :thinking:

Reply:
Oh, I understand now, you're not copying `decidim` sources at all, but just installing the gem via `gem install decidim` :), that's why!